src/ARCtrl/WebRequest/WebRequest.Py.fs

```diff
@@ -0,0 +1,117 @@
+﻿module ARCtrl.WebRequestHelpers.Py
+
+open Fable.Core
+
+// Taken from https://github.com/Zaid-Ajaj/Fable.Requests/blob/master/src/Library.fs
+// Can be replaced either with Fable.SimpleHttp when it supports Python
+// or Fable.Requests when it supports netstandard2.0
+
+#if FABLE_COMPILER_PYTHON
+type private InteropResponseType =
+    abstract status_code : int
+    abstract text : string
+    abstract encoding : string
+    abstract headers : obj
+
+[<RequireQualifiedAccess>]
+module Interop = 
```

Comment:
maybe we should do the same for the web request library from Zaid (copy paste the relevant code). Then we could implement the nodejs part in a cleaner way.